### DISCUSSION TITLE
v2rayng使用数据可以上外网，wifi上不了

### DISCUSSION THREAD

**Discussion by RyuseiS** [2dust/v2rayNG#3225] [General]
手机从魅族18s换成了21pro，发现使用数据可以在浏览器上外网，wifi上不了，但是telegram纸飞机不受影响，v2rayng版本是1.1.15，手机上节点测试是可以联通的，在旧手机连wifi也是可以上外网的，请问这怎么解决喔

**Comment by fishpoaem** [comment]
1.检查V2rayNG这个App的WLAN联网权限是否打开，一般是这样的问题
2.如果不行，再检查WLAN设置，看看无线能否上网，有没有配置错误的代理、IP、网关、DNS、子网掩码这些东西
注：如果WLAN无线网络、蜂窝数据网络、蓝牙无线网络无法上网，可以在手机设置里单独对这些网络进行重置

**Reply by RyuseiS** [reply]
更新到1.8.5解决了^_^
